src/search/move_picker.h

```diff
@@ -229,25 +214,29 @@ class MovePicker {
         return std::nullopt;
     }
 
-    void generateCaptureScores(const BitBoard& board)
+    void generateTacticalScores(const BitBoard& board)
     {
 
         for (uint16_t i = 0; i < m_moves.count(); i++) {
-            if (m_moves[i].isCapture()) {
+            if (m_moves[i].isCapture())
                 m_scores[i] = evaluation::SeeSwap::run(board, m_moves[i]);
-            }
+            // Capture-promotions are handled in SEE swap
```

Comment:
Nitpick: don't like comments between if cases when they're not scoped (can easily lead to annoying syntax errors)

Reply:
Good to know :) Now redundant, but I'll keep this in mind going forwards.